src/main/java/com/dubture/jenkins/digitalocean/ComputerLauncher.java

```diff
@@ -128,8 +113,16 @@ public void onClosed(Channel channel, IOException cause) {
 
             successful = true;
         } catch (Exception e) {
+            try {
+                Jenkins.getInstance().removeNode(computer.getNode());
+            } catch (Exception ee) {
+                ee.printStackTrace(logger);
```

Comment:
quite an unusual way since you're not choosing the lvl you want the stacktrace to be logged to (which might be a good idea).